.luacheckrc

```diff
@@ -3,3 +3,7 @@ globals = {
   '_TEST'
 }
 exclude_files = {'./rootfs/etc/nginx/lua/test/**/*.lua'}
+
+std = {
+  read_globals = { math = { fields = { "randomseed" } } }
+}
```

Comment:
I think this file needs to be rewritten to something like:
```
std = {
  globals = {'_TEST', 'ngx_lua'},
  read_globals = { math = { fields = { "randomseed" } } }
}
```

https://luacheck.readthedocs.io/en/stable/config.html

Reply:
@zrdaley it looks like globals, read_globals, std are all same level, so can not be nested like you suggested. I had to use ignore per file.